cim-client-sdk/src/main/java/com/crossoverjie/cim/client/sdk/impl/ClientImpl.java

```diff
@@ -121,6 +121,7 @@ private CompletableFuture<Boolean> doConnectServer() {
         this.userLogin(future).ifPresentOrElse((cimServer) -> {
             this.doConnectServer(cimServer, future);
             this.loginServer();
+            routeManager.fetchOfflineMsgs(conf.getAuth().getUserId());
```

Comment:
It is recommended to move the function for retrieving offline messages to line 90 to make the code logic clearer.